schemas/user.yml

```diff
@@ -56,3 +56,10 @@
 
         Useful if `user.id` or `user.name` contain confidential information and
         cannot be used.
+
+    - name: domain
+      level: extended
+      type: keyword
+      example: domainname
+      description: >
+        Name of user domain, e.g. Active Directory or LDAP domain name.
```

Comment:
Can you adjust the description to the following, please? Note that the two paragraphs will give us a "short" description automatically, here.

"Name of the directory the user is a member of.

For example, an LDAP or Active Directory domain name."

Reply:
Changed it. Update PR in few seconds.